sdcm/tester.py

```diff
@@ -1091,3 +1091,12 @@ def populate_data_parallel(self, size_in_gb, blocking=True, read=False):
                 self.verify_stress_thread(queue=stress)
 
         return write_queue
+
+    @log_run_info
+    def alter_table_to_in_memory(self, key_space_name="keyspace1", table_name="standard1", node=None):
+        if not node:
+            node = self.db_cluster.nodes[0]
+        compaction_strategy = "%s" % {"class": "InMemoryCompactionStrategy"}
```

Comment:
What is the reason to use string formatting here?

Reply:
no special reason :) I can change it if you don't like it

Reply:
No. I just thought may be it's right way to do that. I have no problem with it